pyanaconda/modules/users/installation.py

```diff
@@ -161,3 +163,33 @@ def run(self):
     def _set_ssh_keys(self):
         for key_data in self._ssh_key_data_list:
             users.set_user_ssh_key(key_data.username, key_data.key)
+
+class ConfigureRootPasswordSSHLogin(Task):
+    """Optionally add an override allowing root to login with password over SSH."""
+
+    SSHD_OVERRIDE_DROP_DIR = "etc/systemd/system/sshd.service.d/"
+    OVERRIDE_CONFIG_NAME = "anaconda.conf"
+
+    def __init__(self, sysroot, root_password_ssh_login_allowed):
+        """Create a new root password SSH login configuration task.
+
+        :param str sysroot: a path to the root of the installed system
+        :param bool root_password_ssh_login_allowed: True allows root to login via SSH with password auth.
+                                                     False prevents it by not changing the default OpenSSH
+                                                     behavior
+        """
+        self._sysroot = sysroot
+        self._root_password_ssh_login_allowed = root_password_ssh_login_allowed
+
+    @property
+    def name(self):
+        return "Configure optional root password SSH login"
+
+    def run(self):
+        # TODO: don't instantiate and run this Task if the override is not needed
+        if self._root_password_ssh_login_allowed:
```

Comment:
Please, log a message if the configuration is not allowed.

Reply:
Will do, though ideally (as in the TODO) this Task should just be skipped in the future in such a case.